.github/workflows/auto-label.yml

```
@@ -0,0 +1,95 @@
+          configuration-path: .github/labeler.yml
+          sync-labels: true
+
+      - name: Auto-label based on changed files
+        uses: actions/labeler@v4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          configuration-path: .github/labeler-files.yml
+          sync-labels: true
+
+      - name: Label based on commit type
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)